control-plane-operator/controllers/hostedcontrolplane/kas/auth.go

```diff
@@ -39,13 +45,25 @@ func generateAuthConfig(spec *configv1.AuthenticationSpec) *AuthenticationConfig
 		JWT: []JWTAuthenticator{},
 	}
 	if spec == nil {
-		return config
+		return config, nil
 	}
 	for _, provider := range spec.OIDCProviders {
+		caData := ""
+		if provider.Issuer.CertificateAuthority.Name != "" {
+			ca := &corev1.ConfigMap{}
```

Comment:
will this get triggered when the configmap content is changed?

Reply:
Yes, HostedControlPlane reconcile is triggered on change of the `Issuer.CertificateAuthority.Name`, changes to the configmap contents, and KAS is redeployed on changes to the auth-config file